src/operations/insert.ts

```diff
@@ -131,7 +131,9 @@ export class InsertManyOperation extends AbstractOperation<InsertManyResult> {
     const writeConcern = WriteConcern.fromOptions(options);
     const bulkWriteOperation = new BulkWriteOperation(
       coll,
-      prepareDocs(coll, this.docs, options).map(document => ({ insertOne: { document } })),
+      maybeAddIdToDocuments(coll, this.docs, options).map(document => ({
+        insertOne: { document }
+      })),
```

Comment:
Is this redundant now since bulk will call the maybeAdd on the array of `{insertOne:{document}}`?

Reply:
fixed